tests/handlers/aws/test_integrations.py

```diff
@@ -72,9 +72,10 @@ def setUpClass(cls) -> None:
         lgc = LogstashContainer(es_container=esc)
         cls.logstash = lgc.start()
 
-        lsc = LocalStackContainer(image="localstack/localstack:1.4.0")
+        lsc = LocalStackContainer(image="localstack/localstack:3.0.0")
```

Comment:
Thanks for this!

We could use `3.0.1` here instead. We should have a look at how to automate these changes with dependabot.